app/models/categories_item.rb

```diff
@@ -1,2 +1,4 @@
 class CategoriesItem < ApplicationRecord
+  belongs_to :category
+  has_many :items
 end
```

Comment:
All the changes you have done to `app/models` are not really contributing to the subject of this PR. I would say that we should revert and bring these changes in future PR when you find that you actually need this.